src/library/trackset/setlogfeature.cpp

```diff
@@ -161,12 +181,44 @@ QList<BasePlaylistFeature::IdAndLabel> SetlogFeature::createPlaylistLabels() {
                 playlistTableModel
                         .data(playlistTableModel.index(row, nameColumn))
                         .toString();
-        BasePlaylistFeature::IdAndLabel idAndLabel;
-        idAndLabel.id = id;
-        idAndLabel.label = name;
-        playlistLabels.append(idAndLabel);
+        QDateTime created =
+                playlistTableModel
+                        .data(playlistTableModel.index(row, createdColumn))
+                        .toDateTime();
+
+        if (selected_id == id) {
+            // save index for selection
+            selected_row = row;
+        }
+
+        // Create the TreeItem whose parent is the invisible root item
+        TreeItem* item = new TreeItem(name, id);
```

Comment:
where is this memory freed? you should consider to use a appropriated smart pointer to handle the lifetime of the object.